moveit_planners/chomp/chomp_optimizer_adapter/src/chomp_optimizer_adapter.cpp

```diff
@@ -34,138 +34,138 @@
 
 /* Author: Raghavender Sahdev */
 
-#include <moveit/planning_request_adapter/planning_request_adapter.h>
-#include <moveit/robot_state/conversions.h>
-#include <moveit/trajectory_processing/trajectory_tools.h>
-#include <moveit_msgs/msg/robot_trajectory.hpp>
-#include <class_loader/class_loader.hpp>
-#include <ros/ros.h>
-
-#include <chomp_motion_planner/chomp_planner.h>
 #include <chomp_motion_planner/chomp_parameters.h>
-#include <moveit/planning_interface/planning_interface.h>
-#include <moveit/trajectory_processing/iterative_time_parameterization.h>
-
+#include <chomp_motion_planner/chomp_planner.h>
 #include <moveit/collision_distance_field/collision_detector_allocator_hybrid.h>
-
+#include <moveit/planning_interface/planning_interface.h>
+#include <moveit/planning_request_adapter/planning_request_adapter.h>
 #include <moveit/robot_state/conversions.h>
+#include <moveit/trajectory_processing/iterative_time_parameterization.h>
+#include <moveit/trajectory_processing/trajectory_tools.h>
 
-#include <vector>
+#include <class_loader/class_loader.hpp>
```

Comment:
Better use `<pluginlib/class_loader.hpp>` (I know, it's not fully consistent in MoveIt, yet)
```suggestion
#include <pluginlib/class_loader.hpp>
```